lib/get-values-from-ci.js

```diff
@@ -0,0 +1,15 @@
+const _ = require('lodash')
+
+const integrationImports = [
+  require('./ci-integrations/travis-ci'),
```

Comment:
Would love to have this filesystem based (`fs.readdirSync()`), as I hate manually maintaining such lists :)